querybook/webapp/components/DataDocDAGExporter/DataDocDAGExporterButton.tsx

```diff
@@ -24,12 +24,11 @@ export const DataDocDAGExporterButton: React.FunctionComponent<IProps> = ({
                 title="Exporter"
             />
             {showModal ? (
-                <Modal
-                    onHide={() => setShowModal(false)}
-                    title="DAG Exporter"
-                    type="fullscreen"
-                >
-                    <DataDocDAGExporter docId={docId} />
+                <Modal onHide={() => setShowModal(false)} type="fullscreen">
```

Comment:
if the purpose of having onClose passed to DataDocDAGExporter is so that it can be blocked when unsaved, i think Modal onHide should be a NOOP function